qiskit_experiments/library/quantum_volume/qv_experiment.py

```diff
@@ -150,7 +150,7 @@ def _get_ideal_data(self, circuit: QuantumCircuit, **run_options) -> List[float]
 
             state_vector = Statevector(ideal_circuit)
             probabilities = state_vector.probabilities()
-        return probabilities
+        return list(probabilities)
```

Comment:
Why is this being cast to `list`? The doc says it's returned as Numpy array so in any case there is an inconsistency here.

Reply:
It should be casted into a list since the type of `probabilities` was not serializable.
I deleted in the description the end of the output description "(as Numpy array)".